rust/gui-client/src-tauri/src/client/tunnel_wrapper/windows.rs

```diff
@@ -0,0 +1,109 @@
+use anyhow::{anyhow, Context as _, Result};
+use connlib_client_shared::Callbacks;
+use firezone_headless_client::{IpcClientMsg, IpcServerMsg};
+use futures::{SinkExt, Stream};
+use secrecy::{ExposeSecret, SecretString};
+use std::{pin::pin, task::Poll};
+use tokio::{net::windows::named_pipe, sync::mpsc};
+use tokio_util::codec::{Framed, LengthDelimitedCodec};
+
+pub(crate) struct TunnelWrapper {
```

Comment:
`TunnelWrapper` seems a bit like a misnomer since it doesn't seem to wrap any `Tunnel`

Reply:
Yeah how about `IpcClient` since the in-process version will be completely gone?

Reply:
Also I started thinking `platform` is better for platform-specific code than `imp` so I changed that in some places

Reply:
Renamed `tunnel_wrapper::TunnelWrapper` to `ipc::Client`